src/rendering/vcCompass.cpp

```diff
@@ -59,23 +59,18 @@ udResult vcCompass_Destroy(vcAnchor **ppCompass)
   return udR_Success;
 }
 
-udResult vcCompass_Render(vcAnchor *pCompass, vcAnchorStyle anchorStyle, const udDouble4x4 &worldViewProj, const udDouble4 &colour /*= udDouble4::create(1.0, 1.0, 1.0, 1.0)*/)
+bool vcCompass_Render(vcAnchor *pCompass, vcAnchorStyle anchorStyle, const udDouble4x4 &worldViewProj, const udDouble4 &colour /*= udDouble4::create(1.0, 1.0, 1.0, 1.0)*/)
```

Comment:
This seems like a strange choice? Also, `udDouble4::one()` could be used for the default parameter?

Reply:
More consistent with our other rendering operations to return `bool` rather than `udResult`.
Also the comment was wrong, I'll fix it.